Formula/deno.rb

```diff
@@ -0,0 +1,35 @@
+class Deno < Formula
+  desc      "Secure TypeScript runtime on V8"
+  homepage  "https://github.com/denoland/deno/"
+  url       "https://github.com/denoland/deno.git",
+      :tag      => "v0.2.5",
+      :revision => "320577c131c8058ba0065f26119ce1ce18bc23d2"
+
+  bottle do
+  end
+
+  depends_on "ninja" => :build
+  depends_on "node" => :build
+  depends_on "python@2" => :build
+  depends_on "rust" => :build
+
+  def install
+    rm_rf ["node_modules", "buildtools", "build_overrides", "testing", "tools/clang"]
+    system "python", "tools/setup.py"
+    system "python", "tools/build.py", "-C", "target/release"
+    ENV["DENO_BUILD_MODE"] = "release"
+    # system "python", "tools/test.py"
+    bin.install "target/release/deno"
+  end
+
+  test do
+    version_string = shell_output("#{bin}/deno --version")
```

Comment:
If we have a proper test like the one below we don't need to test for the version output.

Reply:
I was thinking it was a good idea to keep to ensure the correct version was installed.